src/brogue/RogueMain.c

```diff
@@ -1233,7 +1245,17 @@ void victory(boolean superVictory) {
     sprintf(buf, "%li", totalValue);
     printString(buf, mapToWindowX(60), min(ROWS-1, i + 1), &lightBlue, &black, dbuf);
 
-    i += 4;
+    funkyFade(dbuf, &white, 0, 120, COLS/2, ROWS/2, true);
```

Comment:
Not particularly fond of this second funkyFade, I think it should just go to the next screen

Reply:
Honestly, without this funkyfade, I don't understand how to clear this dbuf correctly and write my text. I have some strange behaviour. With : clearDisplayBuffer and deletesMessages, my achievements list is not display. Maybe a function to force redraw ? But there is no call before in this function. A idea ?